server/mentionscrawler/models/user.py

```diff
@@ -1,22 +1,28 @@
-from flask_sqlalchemy import SQLAlchemy
 from ..db import db
 
 
 class MentionUser(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     email = db.Column(db.String(50), unique=True, nullable=False)
-    company_name = db.Column(db.String(50), nullable=False)
+    company_names = db.relationship("Company", backref="mention_user", lazy=True)
     password = db.Column(db.String(256), nullable=False)
 
-    def __init__(self, email, company_name, password):
+    def __init__(self, email: str, password: str):
         self.email = email
-        self.company_name = company_name
         self.password = password
 
     def __repr__(self):
-        return "id: {}, name: {} email: {}".format(self.id, self.company_name, self.email)
+        return "id: {}, email: {}".format(self.id, self.email)
 
-    @staticmethod
-    def commit_user(new_user: object):
-        db.session.add(new_user)
-        db.session.commit()
+
+class Company(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(50), nullable=False)
+    mention_user_id = db.Column(db.Integer, db.ForeignKey("mention_user.id"), nullable=False)
```

Comment:
Wouldn't be easier to have the company names represented as a string array in the user table?

Reply:
usually one to many relationships in sql are done through a join (you use a a separate table and join them like I'm doing here)

Not sure how that would work for inserting, I know that postgres does support arrays, but I don't believe most databases do, and this makes it simpler for inserting, since I don't think you can just concatenate onto an existing array when updating a value in the database.

If we were using mongo I definitely would have it as an array.